### PR TITLE
Lnbits compatibility

### PR DESCRIPTION
**Pull request by k9ert**
This PR is based on the initial [branch](https://github.com/21isenough/LightningATM/tree/lnbits-compatibility) from @21isenough which brings connectivity to lnbits.
# Testing this
@AxelHamburch has a nice description on how to do that [here](https://github.com/AxelHamburch/LightningATM/blob/doc_lnbits_funding/docs/guide/we_need_your_help.md). (not yet finished, though)

# short version
* stop the ATM
* change the branch on your raspi like this:
```
git remote add k9ert git@github.com:k9ert/LightningATM.git
git fetch k9ert
git checkout lnbits-compatibility-k9ert
```
* Create a lnbits-wallet in your lnbits-instance of your choice and install the withdraw-extension (https://legend.lnbits.com didn't worked for this when i tested it)
* copy and take a note of the admin-key (unfold the API-Info on the right hand side and copy the key from there)
* change the activewallet to lnbits
```
activewallet = lnbits
```
* Create a new section in your `config.ini` as shown in the `example_config.ini`.
```
[lnbits]
# api credentials
url = https://your.lnbits.instance/api/v1
apikey = 
# One of "invoice" or "lnurlw"
method = lnurlw
# only for lnurlw - millisseconds to redeem the lnurlw
timeout = 90000
```
* Don't choose "invoice" if your ATM doesn't have a camera
* insert the apikey

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -13,11 +13,13 @@
 import lndrest
 import lntxbot
 import qr
+import lnbits
 
 import utils
 import importlib
 from PIL import Image
 
+import lnbits
```
Comment (k9ert) [inline app.py:22]: Should not be in there twice, no?!

File: example_config.ini [hunk 2]
```diff
@@ -68,6 +68,15 @@ verify = on
 url =
 creds =
 
+[lnbits]
+# api credentials
+url = https://legend.lnbits.com/api/v1
```
Comment (k9ert) [inline example_config.ini:73]: I think we should encourage the other lnbits-server.
```suggestion
# api credentials (better use a different lnbits-server - decentralize)
url = https://
legend.lnbits.com/api/v1
```

**Comment by lightisfaster** [comment]
Good job! I can‘t test it now because of holiday. If it works, we can push it to the master.

**Comment by k9ert** [comment]
Any news on this? Any feedback welcome!

**Comment by AxelHamburch** [comment]
I believe it well and running. Have no problems with the new API. The initial dropouts of WoS on legend.lnbits.com were probably caused by an overload of the server. I would merge this PR. Then I adapt the documentation to it. 

What do you say @21isenough

**Comment by AxelHamburch** [comment]
@21isenough what do you say? It's fine for me. We could create a version 0.3.1 with the current version and then merge the PR.

**Comment by AxelHamburch** [comment]
@lightisfaster What do you say?

**Pull request merged**